README.md

```diff
@@ -0,0 +1,318 @@
+- Cluster and subnet UUIDs auto-detected
+- Image names: `Windows-Server-2019`, `Windows-Server-2022`
+
+### Bare Metal (None)
+
+- Uses AWS credentials from `~/.aws/config` and `~/.aws/credentials`
+- Platform detected as "none"
+- Provisions using AWS infrastructure
+
+## Architecture
+
+This project uses a modular architecture with separated concerns:
+
+```
+terraform-windows-provisioner/
+├── byoh.sh                 # Main entry point
+├── lib/                    # Library modules
+│   ├── config.sh          # Configuration loading
+│   ├── credentials.sh     # Credential management
+│   ├── platform.sh        # Platform detection & config
+│   ├── terraform.sh       # Terraform operations
+│   └── validation.sh      # Input validation
+├── configs/               # Configuration files
+│   ├── defaults.conf      # Default values
+│   └── examples/          # Platform-specific examples
+└── platforms/             # Platform-specific Terraform
+    ├── aws/
+    ├── azure/
+    ├── gcp/
+    ├── vsphere/
+    ├── nutanix/
+    └── none/
+```
+
+## Troubleshooting
+
+### Check Cluster Credentials
+
+```bash
+oc get secret -n kube-system
+oc get secret -n openshift-machine-api
+```
+
+### Verify Cluster Status
+
+```bash
+oc get clusterversion
+oc get nodes
+```
+
+### View Terraform State
+
+```bash
+cd /tmp/terraform_byoh/<platform>
+terraform show
+terraform output
+```
+
+### Check ConfigMap
+
+```bash
+# Find WMCO namespace
+oc get deployment --all-namespaces | grep windows-machine-config-operator
+
+# View ConfigMap
+oc get configmap windows-instances -n <wmco-namespace> -o yaml
+```
+
+### Enable Debug Logging
+
+```bash
+export BYOH_LOG_LEVEL=DEBUG
+./byoh.sh apply myapp 2
+```
+
+## Contributing
+
+We welcome contributions! Please see [CONTRIBUTING.md](CONTRIBUTING.md) for details on:
+
+- Code of conduct
+- Development setup
+- Pull request process
+- Testing requirements
+
+## Security
+
+For security issues, please see [SECURITY.md](SECURITY.md).
+
+## License
+
+Apache License 2.0. See [LICENSE](LICENSE) for details.
+
+## Project Status
+
+Version: 1.0.0
+
+This is a production-ready, vendor-neutral project suitable for upstream use.
+
+## Support
+
+- **Issues**: Report bugs or request features via GitHub Issues
+- **Documentation**: See [docs/](docs/) for comprehensive guides
+- **Examples**: See [configs/examples/](configs/examples/) for configuration examples
+
+## Acknowledgments
+
+This project is designed to work seamlessly with:
+- OpenShift Windows Container Support
+- Windows Machine Config Operator (WMCO)
+- Kubernetes Windows node support
+
+## Roadmap
+
+- [ ] Additional cloud platform support
+- [ ] Enhanced monitoring and metrics
+- [ ] Integration tests for all platforms
+- [ ] Helm chart for Kubernetes deployment
+- [ ] Web UI for configuration
+
+---
+
+**Note**: This is a generic, vendor-neutral project with no hardcoded values. All configuration is customizable via configuration files or environment variables.
```

Comment:
Could this go at the beginning of the file, [right after the introductory paragraph](https://github.com/openshift/terraform-windows-provisioner/pull/6/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R4)?